src/DeviceRenderer.js

```diff
@@ -471,14 +474,14 @@ module.exports = class DeviceRenderer {
                     this.videoWrapper.prepend(popup);
                     const addSound = () => {
                         this.video.muted = false;
-                        window.removeEventListener('click', addSound);
-                        window.removeEventListener('touchend', addSound);
+                        this.removeAddSoundClickListener();
+                        this.removeAddSoundTouchListener();
                         this.dispatchEvent('video', {msg: 'sound manually allowed by click'});
                         popup.remove();
                         log.debug('Playing video with sound enabled has been authorized due to user click');
                     };
-                    window.addEventListener('click', addSound, {once: true});
-                    window.addEventListener('touchend', addSound, {once: true});
+                    this.removeAddSoundClickListener = this.addListener(window, 'click', addSound, {once: true});
```

Comment:
Ah yes, a more generic way to bind two events to the same handler would be interesting, especially since we also do it with other events than click/touch! I'll look into this, thanks.